LiteCore/Support/Logging.cc

```diff
@@ -415,10 +416,12 @@ namespace litecore {
         va_end(args);
     }
 
+    // Can't make this function static, it breaks the usage of __printflike.
+    // NOLINTBEGIN(readability-convert-member-functions-to-static)
     // Must have sLogMutex held
     void LogDomain::dylog(LogLevel level, const char* domain, unsigned objRef, const char* fmt, va_list args) {
         auto     obj = getObject(objRef);
-        uint64_t pos = 0;
+        uint64_t pos;
```

Comment:
Why not to initilize?

Reply:
Because the variable is assigned a value in either code path - so it will always be initialized by the time it's used. Initializing here has no effect.